_collections/_information/update-contact-info.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default-md
+title: Update Contact Information
+long_title: Update Your TroopMaster Contact Information
+subtitle: To adapt for changes in the provider of our email blasts and to enable the Troop to communicate by text messages, we ask that you change or add your email addresses and cell phone numbers in TroopMaster. This will allow you and your parents to receive email blasts and texts through our new system. We also want to be sure we have the most up to date home phone numbers. Here are the instructions to do so.
+nav_order: 8
+redirect_from:
+  - contact-info-directions.html
+  - information/contact-info-directions.html
+---
+## Steps
+
+1. Start by heading to [the Troop’s TroopMaster page](https://tmweb.troopmaster.com/mysite/troop370atl).
+1. Log in (if you have never logged in before, the username is in this format: ‘firstname’.’lastname’ , ex: john.smith . Note: The name used is the name of the scout, not the parent). The password, if not previously reset, is the scout’s BSA membership ID (consists of 9 numbers).
+1. In the top nav-bar, click ‘Scouts’. A drop-down menu will appear as shown below. Click on ’Scout Management’. It is the only option provided in the dropdown menu.
+![Step 3 Screenshot](/src/troopmaster-contact-info-instructions/pic1.png)
+1. You will be presented with an entry for your scout profile. Click the blue 'view' button on the right as shown below.
+![Step 4 Screenshot](/src/troopmaster-contact-info-instructions/pic2.png)
+1. A window will appear. Click on the blue ‘personal info’ button.
+![Step 5 Screenshot](/src/troopmaster-contact-info-instructions/pic3.png)
+1. Click ‘edit’ in the top right corner of the window as shown below.
+![Step 6 Screenshot](/src/troopmaster-contact-info-instructions/pic4.png)
+1. Add or update your home phone number as well as the Scout's cell phone number. Also, update the email address to the Scout's current email. After, click the red ‘save’ button in the top right corner of the window as shown below. Then click close to return to the main menu.
+![Step 7 Screenshot](/src/troopmaster-contact-info-instructions/step7.png)
+1. To edit contact information for the Scout's parents, click the blue button for the parent whose contact information you would like to change (you can do this for the other parent later) as shown below.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `8`, was `1`](https://app.codacy.com/app/Troop_370/troop370/pullRequest?prid=4642321)